rotkehlchen/globaldb/handler.py

```diff
@@ -334,6 +338,7 @@ def search_assets(filter_query: 'AssetsFilterQuery', db: 'DBHandler') -> List[Di
                     'identifier': entry[0],
                     'name': entry[1],
                     'symbol': entry[2],
+                    'chain': ChainID.deserialize_from_db(entry[3]).serialize() if entry[3] is not None else None,  # noqa:E501
```

Comment:
would need to change this a bit and move the if condition above to construct the dict before appending if you are to do what I mentioned above